client/src/main/java/org/apache/rocketmq/client/impl/factory/MQClientInstance.java

```diff
@@ -102,6 +102,8 @@
         new ConcurrentHashMap<String, HashMap<Long, String>>();
     private final ConcurrentMap<String/* Broker Name */, HashMap<String/* address */, Integer>> brokerVersionTable =
         new ConcurrentHashMap<String, HashMap<String, Integer>>();
+    private final ConcurrentMap<String/* Broker Name */, HashMap<String/* address */, Long>> versionUpdateTimeTable =
```

Comment:
It seems that the versionUpdateTimeTable and  brokerVersionTable  would be difficult to keep insistence.